lib/trino-orc/src/test/java/io/trino/orc/OrcTester.java

```diff
@@ -1215,43 +1216,87 @@ private static Object preprocessWriteValueHive(Type type, Object value)
     static RecordWriter createOrcRecordWriter(File outputFile, Format format, CompressionKind compression, Type type)
             throws IOException
     {
-        JobConf jobConf = new JobConf(newEmptyConfiguration());
-        OrcConf.WRITE_FORMAT.setString(jobConf, format == ORC_12 ? "0.12" : "0.11");
-        OrcConf.COMPRESS.setString(jobConf, compression.name());
-
-        return new OrcOutputFormat().getHiveRecordWriter(
-                jobConf,
-                new Path(outputFile.toURI()),
-                Text.class,
-                compression != NONE,
-                createTableProperties("test", getJavaObjectInspector(type).getTypeName()),
-                () -> {});
+        StandardStructObjectInspector objectInspector = getStandardStructObjectInspector(ImmutableList.of("test"), ImmutableList.of(getJavaObjectInspector(type)));
+
+        return RecordWriterBuilder.builder(outputFile, format, compression)
+                .putColumns(objectInspector)
+                .withBloomFilter(objectInspector, 0.50)
```

Comment:
This seems to be an addition compared to the previous version of the method. 
I see in the next commit that you remove the method `createTableProperties` - the two commits are probably related.

Reply:
`Remove unused OrcTester#createTableProperties` squashed to `Add builder for ORC RecordWriter`